src/opcua-iiot-result-filter.ts

```diff
@@ -158,7 +163,22 @@ module.exports = (RED: nodered.NodeAPI) => {
       return result
     }
 
-    const convertResult = (msg: FilterInputPayload, result: Todo) => {
+    const convertAllResults = (payload: FilterInputPayload, result: Todo | Todo[]) => {
+      if (!isArray(result)) {
```

Comment:
Currently `isArray` is just a wrapper of `Array.isArray()` right? TypeScript should also be able to defer the type after using the normale Array.isArray:

```ts
declare const foo: string | string[];
if (Array.isArray(foo)) {
  // foo is string[]
} else {
  // foo is string
}
```

What is the purpose of `isArray`

Reply:
It looks like the difference is that isArray can be called with a generic, asserting a the type of array object. Potentially useful, but not in this case.